CDS_Dashboard/script.js

```diff
@@ -0,0 +1,552 @@
+                objective: 'Fueling Economic Growth',
+                picture: 'bureauLogos/DOC.png'
+            }, {
+                y: 1, 
+                name: 'DOC',
+                color: '#0057a9',
+                initiative: 'White House Council of Women and Girls',
+                services: 'Data Science, UX/UI',
+                objective: 'Delivering Data Services',
+                picture: 'bureauLogos/DOC.png'
+            }]
+        }
+    }];  // end of data property
+
+function setChart(options) {
+    chart.setTitle({
+        text: options.title
+    }, {
+        text: options.subtitle
+    });
+    chart.series[0].remove(false); //boolean decides if it needs to redraw immediately, sinc we are doing more we do not set it to true, we remove the series
+    chart.xAxis[0].setCategories(options.categories, false);
+    chart.addSeries({
+        type: options.type,
+        name: options.name,
+        data: options.data,
+        color: options.color || 'white'
+    }, false); // adds the series (which contains the data) - we pass it a new object and tell it false so it doesn't automatically re-render
+     // xAxis[0] since there is only 1 axis - setCategory sets categories from the array you pass it (options.categories - you use false so it doesn't automatically redraw THUS you use chart.redraw() after)
+    chart.redraw() // redraws chart
+}
+
+var secondChart = $('#secondContainer'); 
+var firstChart = $('#chartContainer');
+
+function moveChart() {
+    if (mq.matches) {
+        return 0
+    } else if (desktopMq.matches) {
+        firstChart.animate({'marginLeft' : '-=200px'}, 100);
+        firstChart.css('display', 'inline-block');
+    } else if (desktopMq2) {
+        firstChart.animate({'marginLeft' : "-=250px"}, 100);
+        firstChart.css("display", "inline-block");
+
+        console.log('moveChart running');
+    } else {
+        firstChart.animate({'marginLeft' : "-=300px"}, 100);
+        firstChart.css("display", "inline-block");
+        console.log('moveChart running');
+    }   
+}
+
+function setSecondChart(options) {
+    chart2.setTitle({
+        text: options.title
+    }, {
+        text: options.subtitle
+    });
+    chart2.series[0].remove(false); //boolean decides if it needs to redraw immediately, sinc we are doing more we do not set it to true, we remove the series
+    chart2.xAxis[0].setCategories(options.categories, false);
+    chart2.addSeries({
+        type: options.type,
+        name: options.name,
+        data: options.data,
+        color: options.color || 'white'
+    }, false); // adds the series (which contains the data) - we pass it a new object and tell it false so it doesn't automatically re-render
+     // xAxis[0] since there is only 1 axis - setCategory sets categories from the array you pass it (options.categories - you use false so it doesn't automatically redraw THUS you use chart.redraw() after)
+    chart2.redraw() // redraws chart
```

Comment:
Semicolon.

I'd suggest running this file through a JS linter. Come to me if you need help with that.